.circleci/config.yml

```diff
@@ -227,10 +227,10 @@ jobs:
           keys:
             - deps-{{ .Branch }}-{{ checksum ".pnp.cjs" }}
       - run: yarn install --immutable-cache
+      - run: yarn run build
       - run: yarn eslint --max-warnings=0 .
-      - run: yarn test --coverage --maxWorkers=2
+      - run: yarn workspaces foreach --exclude terra-integration-tests run test --coverage --maxWorkers=2 --passWithNoTests
       - run: yarn check-types
-      - run: DISABLE_ESLINT_PLUGIN=true yarn build # already linted above
```

Comment:
ESLint and tests require the packages to be built first so that imports from the packages can be resolved.